bedrock/security/templates/security/bug-bounty.html

```diff
@@ -31,6 +31,8 @@ <h2 id="general-eligibility">General Eligibility</h2>
     <li>You must not be an employee, contractor, or otherwise have a business relationship with the Mozilla Foundation or any of its subsidiaries.</li>
     <li>You should use your best effort not to access, modify, delete, or store user data or Mozilla’s data. Instead, use your own accounts or test accounts for security research purposes.</li>
     <li>If you inadvertently access, modify, delete, or store user data, we ask that you notify Mozilla immediately at <a href="mailto:security@mozilla.org">security@mozilla.org</a> and delete any stored data after notifying us.</li>
+    <li>You should also use your best effort not to harm the availability or stability of our services, for example, by running aggressive scanning of those services. Instead, use a local development instance of the service that you want to test. </li>
+    <li>Whenever it is explicitly stated in our program scope, you are expected to test on the provided instances instead of production.</li>
```

Comment:
Shall we provide an "e.g. staging" addition here to firmly avoid misunderstandings?